### PR TITLE
application: serial_lte_modem: Support nRF9131 EK

### PR DESCRIPTION
**Pull request by junqingzou**
Add the overlay files to support nRF9131 EK.
NOTE UART_2 cannot be used as I2C_2 is in use on the EK.

### CODE REVIEW

File: applications/serial_lte_modem/boards/nrf9131ek_nrf9131_ns.conf
Comment (MarkusLassila) [inline applications/serial_lte_modem/boards/nrf9131ek_nrf9131_ns.conf:11]: These comments should be removed as they apply to a scenario where UART2 is used. Which is, as far as I understand, not a scenario with this board.

File: applications/serial_lte_modem/boards/nrf9131ek_nrf9131_ns.overlay
Comment (tomi-font) [inline applications/serial_lte_modem/boards/nrf9131ek_nrf9131_ns.overlay:1]: I'm pondering whether it would make sense that we use a common overlay file for this. Not sure.

Comment (junqingzou) [reply applications/serial_lte_modem/boards/nrf9131ek_nrf9131_ns.overlay:1]: how?

Comment (tomi-font) [reply applications/serial_lte_modem/boards/nrf9131ek_nrf9131_ns.overlay:1]: By having the contents of this file be in a generic `*.overlay` file, that others like this one `#include`. @MarkusLassila An opinion on this?

Comment (MarkusLassila) [reply applications/serial_lte_modem/boards/nrf9131ek_nrf9131_ns.overlay:1]: You mean that the chosen and uart0 parts from the overlay would be in common file as they are repeated over multiple files? I mean, maybe. It is repetitive to have them multiple times, but it is a bit simpler for someone who is looking at these for the first time if they are duplicated. I do not think that it needs to be solved in this PR though.

**Review by MarkusLassila — APPROVED** [review]

File: applications/serial_lte_modem/boards/nrf9131ek_nrf9131_ns.overlay
Comment (tomi-font) [reply applications/serial_lte_modem/boards/nrf9131ek_nrf9131_ns.overlay:1]: Yes. Agreed.

**Review by tomi-font — APPROVED** [review]

**Comment by junqingzou** [comment]
@SeppoTakalo pls help to review in time.

**Comment by tomi-font** [comment]
I think that you can ask for this PR to be merged already as it has two code owner approvals.

**Review by SeppoTakalo — APPROVED** [review]

**Pull request merged**
